package.json

```diff
@@ -8,27 +8,34 @@
     "lint-js": "eslint static/js",
     "test-python": "python3 -m unittest discover tests",
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "watch": "watch -p static/js/**/*.js -c 'webpack --env.development' & watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
+    "watch-css": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
+    "watch-js": "watch -p static/js/**/*.js -c 'webpack --env.development'",
     "serve": "./entrypoint 0.0.0.0:${PORT}",
     "build-css": "node-sass --include-path node_modules static/sass --output-style compressed --output static/css && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",
-    "build": "mkdir -p static/js && webpack && yarn run build-css"
+    "build": "mkdir -p static/js && webpack && yarn run build-css",
+    "start": "yarn run build && concurrently --raw 'yarn run watch-css' 'yarn run watch-js' 'yarn run serve'",
```

Comment:
Add `--kill-others` to stop processes if one fails, so we can catch errors in the Github checks. See [https://github.com/canonical-web-and-design/ubuntu.com/pull/7700/files](here)